tests/integration/models/release.spec.ts

```diff
@@ -263,7 +263,7 @@ describe('Release Model', function () {
 		});
 
 		describe(`given ${uniquePropertyNames.length} draft releases`, function () {
-			const testReleaseByField: Dictionary<BalenaSdk.Release> = {};
+			const testReleaseByField: Dictionary<BalenaSdk.Release['Read']> = {};
```

Comment:
If you do v
```suggestion
			const testReleaseByField: Dictionary<PickDeferred<BalenaSdk.Release['Read'],
				| 'id'
				| 'commit'
				| 'raw_version'
				| 'belongs_to__application'
			>> = {};
```
then we can drop the other two `as BalenaSdk.Release['Read']` casts that were added.